cli/resource.go

```diff
@@ -24,7 +26,24 @@ func runResource(args *docopt.Args, client *controller.Client) error {
 	if args.Bool["add"] {
 		return runResourceAdd(args, client)
 	}
-	return fmt.Errorf("Top-level command not implemented.")
+
+	resources, err := client.AppResourceList(mustApp())
+	if err != nil {
+		return err
+	}
+
+	w := tabWriter()
+	defer w.Flush()
+
+	var provider *ct.Provider
+
+	listRec(w, "ID", "ProviderID", "ProviderName")
```

Comment:
I think we should add spaces between Provider and ID/Name, so `Provider ID`, etc.